src/routes/Private.jsx

```diff
@@ -18,6 +18,9 @@ import DataExplorerView from '../components/search/DataExplorerView';
 const styles = (theme) => ({
   wrapper: {
     fontFamily: theme.typography.fontFamily,
+    height: 'calc(100% - 64px)',
```

Comment:
This looks pretty unusual!

Reply:
What is our browser support matrix?  This will work on modern browsers (and is admittedly, super handy) but won't work on ...sigh...older IE.

Also, it strikes me as strange to be doing layout stuff in a class that seems dedicated to routing

Reply:
pushing this to a new ticket to fix the underlying issue!